addons/zylann.hterrain/hterrain_detail_layer.gd

```diff
@@ -223,6 +224,21 @@ func get_density() -> float:
 	return density
 
 
+func set_billboard_scale(v: int):
+	if billboard_scale == v:
+		return
+	billboard_scale = v
+	
+	_multimesh = _generate_multimesh(CHUNK_SIZE, density, null, billboard_scale)
```

Comment:
As said in another review, I believe this script will need to defer the creation of the multimesh, because it's going to generate again for every property being set by the scene loader https://github.com/Zylann/godot_heightmap_plugin/pull/126/files#r396803137